Cartfile.resolved

```diff
@@ -1 +1 @@
-github "CareEvolution/MarkdownAttributedString" "1.0.2"
+github "CareEvolution/MarkdownAttributedString" "1.0.3"
```

Comment:
Release 1.0.3 of MarkdownAttributedString was made before merging https://github.com/CareEvolution/MarkdownAttributedString/pull/4 — guessing that wasn't intentional